pyproject.toml

```diff
@@ -34,14 +35,17 @@ dependencies = [
 [project.optional-dependencies]
 dev = [
     "qiskit-addon-sqd[test,nbtest,lint,docs]",
+    "tox>=4.4.3",
 ]
-test = [
-    "tox>=4.0",
```

Comment:
As discussed internally on the `qiskit-addon-utils` package, we do not need tox to get installed inside of each tox environment itself.